welcome/oke_about.adoc

```diff
@@ -313,7 +313,7 @@ s| Feature s| {oke} s| {product-title} s| Operator name
 | Kourier Ingress Controller | Not included | Included | N/A
 | RHT Middleware Bundles Sub Compatibility (not included in {product-title}) | Not included | Included | N/A
 | IBM Cloud Pak Sub Compatibility (not included in {product-title}) | Not included | Included | N/A
-| OpenShift Do (`odo`) | Not included | Included | N/A
+// | OpenShift Do (`odo`) | Not included | Included | N/A
```

Comment:
Same comment.

Reply:
@valaparthvi I have also removed the comment from Line 225 from this "oke_about.adoc" file